posthog/api/utils.py

```diff
@@ -333,7 +356,11 @@ def safe_clickhouse_string(s: str) -> str:
 
 
 def create_event_definitions_sql(
-    event_type: EventDefinitionType, is_enterprise: bool = False, conditions: str = ""
+    event_type: EventDefinitionType,
+    is_enterprise: bool = False,
+    conditions: str = "",
+    order_UNSAFE: str = "",
+    order_direction: str = "DESC",
```

Comment:
`order_UNSAFE` and `order_direction` is kind of confusing. I mean, "order" is more general than "order_direction", yet both are here at the same time… and both are equally unsafe. BTW the correct-er type for `order_direction` would be `Literal["DESC", "ASC"]`